src/templates/organization.js

```diff
@@ -13,41 +13,34 @@ import OrganizationSocial from "../components/OrganizationSocial"
 import Layout from "../components/layout"
 import SEO from "../components/seo"
 import Section from "../components/Section"
+import { transformOrganization } from "../utils/airtable"
 
-const OrganizationTemplate = ({ data }) => {
+export default function OrganizationTemplate({ data }) {
   const siteTitle = data.site.siteMetadata.title
-  const orgData = data.airtable.data
+  const org = transformOrganization(data.organization)
+  const topCategories = org.categories.filter(cat => !cat.parent)
+  const subCategories = org.categories.filter(cat => cat.parent)
+  const topCategory = topCategories[0] || subCategories[0].parent
 
-  const sector = orgData.Sector && orgData.Sector[0]?.data
-  const org = {
-    logo: orgData.Logo?.localFiles?.[0]?.childImageSharp?.fixed,
-    title: orgData.Name,
-    sector: sector && {
-      name: sector.Name,
-      slug: sector.Slug,
-    },
-    tagline: orgData.Tagline,
-    about: orgData.About && orgData.About.replace(orgData.Tagline, ""),
-    location: orgData.HQ_Location,
-    headcount: orgData.Headcount,
-    orgType: orgData.Organization_Type,
-    homepage: orgData.Homepage,
-    linkedIn: orgData.LinkedIn,
-    twitter: orgData.Twitter,
-    tags: orgData.Tags,
-  }
+  // Show all sub categories as well as top categories not represented
+  // by sub categories
+  const categoryList = topCategories
+    .filter(
+      category => !subCategories.map(cat => cat.parent.id).includes(category.id)
+    )
+    .concat(subCategories)
```

Comment:
this logic is a bit funky... but basically i just wanted a nice list of cats, without being redundant.